hooks/ad-join-relation-joined.ps1

```diff
@@ -22,27 +22,28 @@ try {
     Import-Module JujuHooks
     Import-Module JujuUtils
 
+    $renameReboot = Rename-JujuUnit
```

Comment:
juju has gotten quite good at naming units. Can you check if current machine names exceed the 15 character netbios limit? I would love to drop this "feature"